src/ssvc/decision_points/public_safety_impact.py

```diff
@@ -96,14 +97,25 @@
     ),
 )
 
+PUBLIC_SAFETY_IMPACT_2_0_1 = SsvcDecisionPoint(
+    name="Public Safety Impact",
+    description="A coarse-grained representation of impact to public safety.",
+    key="PSI",
+    version="2.0.1",
+    values=(
+        MINIMAL_3,
+        SIGNIFICANT_1,
+    ),
+)
+
 
 def main():
-    print_versions_and_diffs(
-        [
-            PUBLIC_WELL_BEING_IMPACT_1,
-            PUBLIC_SAFETY_IMPACT_2,
-        ]
+    versions = (
+        PUBLIC_WELL_BEING_IMPACT_1,
+        PUBLIC_SAFETY_IMPACT_2,
+        PUBLIC_SAFETY_IMPACT_2_0_1,
     )
+    print_versions_and_diffs(versions)
```

Comment:
Running this through print_versions_and_diffs confirmed it deserved a new patch version

```
Public Safety Impact v2.0.1 description did not change
Public Safety Impact v2.0.1 key did not change
Public Safety Impact v2.0.1 version changed from 2.0.0
Public Safety Impact v2.0.1 value names did not change
Public Safety Impact v2.0.1 value Minimal key did not change
Public Safety Impact v2.0.1 value Significant key did not change
(patch) Public Safety Impact v2.0.1 value Minimal description changed
(patch) Public Safety Impact v2.0.1 value Significant description changed
Public Safety Impact v2.0.1 appears to be a patch change
```